radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenClassReference.java

```diff
@@ -1,22 +1,17 @@
-package com.radixdlt.client.atommodel.tokens;
+package com.radixdlt.client.application.translate.tokens;
```

Comment:
I get why `RadixResourceIdentifier` is in application, but I don't understand why `TokenClassReference` is in `application`, not `atommodel`. It sounds like an `atommodel`-level aspect to me.

Reply:
RRI is actually the only thing that is `atommodel` level at the moment since that is the input into quarks. `TokenClassReference` on the other hand isn't used in the atom model and is just a helper to map address + symbol to address + type + name.